queue/remote_base.go

```diff
@@ -251,40 +252,33 @@ func (q *remoteBase) Complete(ctx context.Context, j amboy.Job) error {
 
 			err = q.driver.Complete(ctx, j)
 			if err != nil {
-				if time.Since(startAt) > time.Minute+q.Info().LockTimeout {
```

Comment:
Is this version of looping specifically keeping our jobs  < a minute? Do we just not think this is going to be a concern, since even large generators seem to hit 10 retries? Or is this if statement just not doing anything?

Reply:
It's not doing anything, it's just old logging that doesn't really serve a purpose.